.hgignore

```diff
@@ -0,0 +1,30 @@
+^\.servo
+^\.cargo
+^\.servobuild
+^target/
+^ports/android/bin
+^ports/android/libs
+^ports/android/local.properties
+^ports/android/obj
+^python/_virtualenv
+^python/tidy/servo_tidy\.egg-info
+~$
+\.pkl$
+\.pyc$
+\.swp$
+\.swo$
+\.csv$
```

Comment:
Should these be `*.` instead of `\.` since they're supposed to be wildcards for that extension, rather than an escaped `.` for a hidden directory/file? Apologies if I'm misreading the Mercurical docs...